.changeset/four-pots-beam.md

```diff
@@ -0,0 +1,6 @@
+---
+"@finos/legend-studio": minor
+"@finos/legend-studio-app": minor
+---
+
+Add Legend Studio extension for external language Morphir.
```

Comment:
This message should belong to only `@finos/legend-studio-app` technically.

Reply:
Created separate changesets.